components/formats-gpl/src/loci/formats/in/UnisokuReader.java

```diff
@@ -206,7 +211,7 @@ else if (key.startsWith(":y_data ->")) {
             pixelSizeY = Double.parseDouble(v[2]) - Double.parseDouble(v[1]);
             pixelSizeY /= getSizeY();
             if (unit.equals("nm")) {
-              pixelSizeY /= 1000;
+              pixelSizeYUnit = UNITS.NM;
```

Comment:
Happy to use `FormatTools.getPhysicalSizeX(value, unit)` as in https://github.com/dgault/bioformats/commit/5019659229e5e496b77b67cae53f8a7ed6ce28f3 if is sensible.